.github/workflows/auto-update-keptn-spec.yml

```diff
@@ -0,0 +1,45 @@
+name: Auto Update - Spec
+on:
+  repository_dispatch:
+    types: [spec-update]
+defaults:
+  run:
+    shell: bash
+env:
+  KEPTN_BOT_USER: "keptn-bot <86361500+keptn-bot@users.noreply.github.com>"
+jobs:
+  update-spec:
+    env:
+      SPEC_REF: ${{ github.event.client_payload.ref }}
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Extract spec tag
+        run: echo "SPEC_TAG=${SPEC_REF#refs/tags/}" >> $GITHUB_ENV
+
+      - name: Update keptn/spec
+        env:
+          SPEC_TAG: ${{ env.SPEC_TAG }}
+        run: |
+          echo 'shkeptnspecversion: "$SPEC_TAG"\n' > ./config/config.yaml
```

Comment:
not completely sure, but I think `\n` is printed as string into the file... pls check

Reply:
it works on my machine :)
but actually echo already adds a newline anyways so the additional `\n` can be removed